tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -19,6 +20,8 @@
 
 tf_hub_cache_dir = os.environ.get('TFHUB_CACHE_DIR',
                                   os.path.join(tempfile.gettempdir(), "tfhub_modules"))
+hf_cache_dir = os.environ.get('HF_CACHE_DIR',
+                                  os.path.join(Path.home(), ".cache", "huggingface"))
```

Comment:
```suggestion
                              os.path.join(Path.home(), ".cache", "huggingface"))
```

Reply:
please use `tempfile.gettempdir()` and create `huggingface` dir there

Reply:
Done.